process_meteo_data.py

```diff
@@ -153,12 +142,29 @@ def insert_csv2db(csv_file, cursor, region, datasource_url):
             date = f"{row[0]}-{row[1]}-{row[2].zfill(2)}"
             measured_temperature = update_measurement_format(row[3])
             if measured_temperature == "":
-                print(f"Measured temperature failed at: {meteostation_name}, date: {date} value: ->{measured_temperature}<-")
+                #print(f"Measured temperature failed at: {meteostation_name}, date: {date} value: ->{measured_temperature}<-")
                 continue
-            cursor.execute(f"""INSERT INTO temperatures VALUES 
-                                 (null, {meteostation_id}, "{date}", {update_measurement_format(row[3])} )
-                            """)
+            entry = tuple([location_id, date, update_measurement_format(row[3])])
```

Comment:
tuple lze vytvořit i přímo, bez vytváření dočasného seznamu, stačí hodnoty oddělit čárkami, závorky nejsou povinné, takhle
```python
entry = location_id, date, update_measurement_format(row[3])
```
Zkoušel jsem to měřit a má to dokonce i vliv na rychlost, i když v tomhle případě se to ztratí v ostatních operacích.

Reply:
Tak tu zkouším vytvořit tuple bez seznamu, ale nejde mi to. :( 
Bez závorek mi to dává syntax error. Když tam dám závorky, dostanu TypeError...tak nevím...
mám Python 3.8.6
 in create_list_for_bulk
    entry = tuple (location_id, date, update_measurement_format(row[3]))
TypeError: tuple expected at most 1 argument, got 3

Reply:
Bez seznamu a toho konstruktoru tuple, tak jak jsem to napsal ;-)

Reply:
A jo, tak to se omlouvám, nějak jsem si toho chybějícího konstruktoru včera nevšimla. To je dobrý tip, díky. :)
Už jsem to upravila.